qiskit_nature/second_q/algorithms/initial_points/hf_initial_point.py

```diff
@@ -147,10 +147,18 @@ def compute(
         self._compute()
 
     def _compute(self) -> None:
+        """Computes the HF initial point array for a given excitation list.
+
+        In the Hartree-Fock case this is simply an all-zero array.
+
+        Returns:
+            An all-zero array with the same length as the excitation list.
+        """
         self._parameters = np.zeros(len(self._excitation_list))
 
-    def get_energy(self) -> float:
```

Comment:
Rather than removing this function, should I formally depreciate it in favour of the property?

Reply:
Since this is fully in the new stack, there is no need. But you can write in the release note that this name changed :+1:

Reply:
Perfect. I had already noted the change in the release note.